05_iigo/index.tex

```diff
@@ -477,6 +477,40 @@ \subsection{End of Session Actions}
     \item The Judge has the \emph{power} to decide to hold an election for a new President.
 \end{enumerate}
 
+\section{Executive branch implementation}
+
+The executive branch has been implemented to facilitate the use of powers given to the president. Because any agent could potentially become the president, implementation of the executive branch has been split into two interconnected parts:
```

Comment:
```suggestion
The executive branch has been implemented to facilitate the use of powers given to the President. Because any agent could potentially become the President, the implementation of the executive branch has been split into two interconnected parts:
```